onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -276,6 +275,13 @@ template <typename T>
 Attention<T>::Attention(const OpKernelInfo& info) : OpKernel(info), AttentionCPUBase(info) {
 }
 
+template <typename T>
+void Attention<T>::FreePackedWeights() {
```

Comment:
Looks good. Very minor nit: Is it enough for this method to just be a `static` method in this file and not a member of the class ? The method can take-in the `BufferUniquePtr`s (passed in by the caller) and reset them. Does that provide some binary size savings because this function need not be duplicated for every template parameter T ?

Reply:
updated